modules/distr-tracing-rn-fixed-issues.adoc

```diff
@@ -14,7 +14,21 @@ Fix - What did we change to fix the problem?
 Result - How has the behavior changed as a result? Try to avoid “It is fixed” or “The issue is resolved” or “The error no longer presents”.
 ////
 
-* link:https://issues.redhat.com/browse/TRACING-2337[TRACING-2337] Jaeger is logging a repetitive warning message in the Jaeger logs similar to the following:
+* link:https://issues.redhat.com/browse/OSSM-1910[OSSM-1910]
+Because of an issue introduced in version 2.6, TLS connections could not be established with {product-title} {SMProductShortName}. 
+This update resolves the issue by changing the service port names to match conventions used by {product-title} {SMProductShortName} and Istio. 
+
+
+* link:https://issues.redhat.com/browse/OBSDA-208[OBSDA-208]
+ Before this update, the default 200m CPU and 256Mi memory resource limits  could cause {OTELShortName} to get reinstalled on large clusters.
```

Comment:
to get continuously restarted

Reply:
Revised per you suggestion to read, `Before this update, the default 200m CPU and 256Mi memory resource limits could cause {OTELShortName} to restart continuously on large clusters.`